bin.js

```diff
@@ -271,11 +271,9 @@ function build (entry, outputDir, argv, done) {
     if (argv.uglify !== false && argv.uglify !== 'false') {
       log.debug('uglify starting')
       js = uglify.minify(js, {
-        fromString: true,
-        compress: true,
-        mangle: true,
-        filename: file,
-        sourceMaps: false
+        sourceMap: {
+          filename: file
```

Comment:
I'm not sure we always want source maps enabled; we should enable stuff like `unsafe` and mangle attributes tho :sparkles: